lms/templates/courseware/progress.html

```diff
@@ -78,6 +78,26 @@ <h4 class="hd hd-4 title">${certificate_data.title}</h4>
                 </div>
                 %endif
 
+                %if upgrade_available:
+                <div class="upgrade-banner">
+                    <div class="notification-color-border"></div>
+                    <div class="notification-content">
+                      <div class="upgrade-icon">
+                        <img src="${STATIC_URL}images/edx-verified-mini-cert.png">
```

Comment:
This isn't the right way to get the static URL for the image, because it won't include the MD5 hash, and hence it won't be cached. You should use this instead:

    ${static.url('images/edx-verified-mini-cert.png')}